src/Payments.Mvc/ClientApp/css/_regions.scss

```diff
@@ -349,7 +392,8 @@ table.invoice-details-table {
                 color: $primary-font;
 
                 &:nth-child(3),
-                &:nth-child(4) {
+                &:nth-child(4),
+                &:nth-child(5) {
```

Comment:
that's a whole pile of index specific styles

Reply:
Just following the code style left by the last guy...